web/app/scripts/src/test.es.js

```diff
@@ -0,0 +1,83 @@
+const apiDesc = {
```

Comment:
Can this be in a separate file? And is it not the same as the current api description json file(s)?